lib/3scale/backend/stats/key_generator.rb

```diff
@@ -4,7 +4,7 @@ module Stats
       class KeyGenerator
         attr_reader :service_id, :applications, :metrics, :users, :from, :to
 
-        def initialize(service_id:, applications: [], metrics: [], users: [], from:, to:)
+        def initialize(service_id:, applications: [], metrics: [], users: [], from:, to:, **)
```

Comment:
Why do we need `**` here?

Reply:
`context_info` is passed as well. But I will check it required.

Reply:
not required now. But `DeleteJobDef` has `context_info` to be used here
```ruby
Resque.enqueue(PartitionGeneratorJob, Time.now.getutc.to_f, service_id, applications,
               metrics, users, from, to, context_info)                               
```